.coveragerc

```diff
@@ -0,0 +1,8 @@
+[run]
+branch = true
+omit =
+    */__init__.py
+
+[report]
+show_missing = true
+fail_under = 90
```

Comment:
From previous runs, it looks like we should be targeting 82% https://pantheon.corp.google.com/cloud-build/builds;region=global/a1af37f6-3b62-4dec-868f-c93db231ae89;step=2?e=13802955&mods=-autopush_coliseum&project=langchain-spanner-testing

Reply:
Yes the current code coverage is 81.80%. I updated the threshold to 81%